typescript-definitions-derive/src/tests.rs

```diff
@@ -185,7 +185,86 @@ mod macro_test {
             }
         );
         assert_conversion!(tokens, "/**\n * a well-documented struct\n */\n\
-            export type WellDocumented = { /**\n * even the field is documented\n */ well_documented: string }");
+            export type WellDocumented = { /**\n * even the field is documented\n */\n well_documented: string }");
+    }
+
+    #[test]
+    fn doc_c_like_enum_variant() {
```

Comment:
**Not-blocking**

Nice set of tests! It does make me wonder if we should compare against assets/fixtures in some cases to improve readability. Parsing all the line breaks on a large struct is not the easiest.

Reply:
We can do this in a followup.